3.3/.sti/bin/assemble

```diff
@@ -14,3 +14,12 @@ if [ -f setup.py ]; then
 elif [ -f requirements.txt ]; then
     pip install --user -r requirements.txt
 fi
+
+# set permissions for any installed artifacts
+chmod -R og+rwx /opt/openshift
+
+# remove pip temporary directory
+pip_tmp=/tmp/pip_build_default
+if [ -d $pip_tmp ]; then
+    rm -rf $pip_tmp
+fi
```

Comment:
When the application starts, under a different user, pip complains that the /tmp/pip_build_default directory already exists and is not owned by the current user. The app seems to keep on working... it will serve html, but I didn't want potentially other function to be broken because of it.